docs/src/details/integration/building/cmake.rst

```diff
@@ -70,11 +77,27 @@ If you use cmake to install lvgl 3 folders will be created.
 
 Build with Command line
 -----------------------
+You can also build your project using the command line. Run the following commands to use the presets:
+
+.. code-block:: bash
+
+    cmake --preset windows-base
```

Comment:
Please describe in 1-2 sentences what is a CMake preset. What other common presets are available?